renderdoc/android/android_patch.cpp

```diff
@@ -572,7 +584,17 @@ extern "C" RENDERDOC_API AndroidFlags RENDERDOC_CC RENDERDOC_MakeDebuggablePacka
 
   // Try the following steps, bailing if anything fails
   if(!Android::PullAPK(deviceID, pkgPath, origAPK))
-    return AndroidFlags::ManifestPatchFailure;
+  {
+    // Copy the APK to public storage, then try to pull again
+    std::string copyPath = "/storage/emulated/0/Download/" + package + ".copy.apk";
```

Comment:
Is it guaranteed that `/storage/emulated/0/Download/` is always present on all devices? This only happens in the failure case so it's not too bad, but I'm wary of relying on anything to behave consistently on android.

Reply:
I have modified the `copyPath` to **sdcard** (**sdcard** is a symbolic link to the primary external storage on Android 6.0+ and it's also used in the `GetTempRootPath()` method of the **android_stringio.cpp**).